src/link/apm32_flash_f4_split.ld

```diff
@@ -0,0 +1,216 @@
+/*
```

Comment:
Yes, this file is intentionally kept separate from the STM32F4 series to ensure the independence and modularity of each microcontroller family. While the functionality may be similar, maintaining separate files allows for tailored optimizations and configurations specific to the unique characteristics of each MCU series. This approach enhances maintainability and scalability of the codebase.

Reply:
One problem is that codebase is large and it is easy fix problem for one architecture, but forget to apply fix to other architectures too (UART implementation is good example).
Another problem is that the files will slowly drift apart. In future, it will get harder to apply fix to all versions.